imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -1172,9 +1190,60 @@ func Test_fdReaddir(t *testing.T) {
 				}
 			},
 			buf: 0, bufLen: 30, // length is longer than the second entry
-			cookie:          2,  // previous d_next
-			expectedBufused: 30, // length to read exactly one more.
-			expectedMem:     append(dirent2, dirent3[0:4]...),
+			cookie:          1,  // previous d_next
```

Comment:
I have changed this to 1 because if you send in cookie 1, you expect to get entry 2.